apps/balancer-tools/src/components/Table.tsx

```diff
@@ -109,6 +116,43 @@ function Body({ children }: React.PropsWithChildren) {
   return <tbody>{children}</tbody>;
 }
 
+function BodyCellLink({
+  children,
+  href,
+  linkClassNames,
+  tdClassNames,
```

Comment:
Maybe? Originally this component was only on `PoolTableList.tsx`, I moved to there because whenever we want make a `td` that is also a Link we'll always to in this same way. If you think this is a shouldn't be here I wouldn't mind moving it back.

Reply:
nah, not worth the additional effort just more of a remark from my side